### PR TITLE
Feature/currency selector

### PR DESCRIPTION
**Pull request by TSherpa1**

### CODE REVIEW

File: src/app/components/homeComponents/currencySelector/page.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import { fetchCoinList } from "@/lib/features/coinListSlice";
+import Slider from "react-slick";
+import CurrencyDiv from "./currencySelectorComponents/CurrencyDiv";
+import {
+  PrevArrow,
+  NextArrow,
+} from "./currencySelectorComponents/SliderArrows";
+
+const CurrencySelector = () => {
+  //*This code below is commented out to avoid extra api calls, using the dummy data below it
```
Comment (Basiol97) [inline src/app/components/homeComponents/currencySelector/page.tsx]: You can simply delete it

File: src/app/components/homeComponents/currencySelector/currencySelectorComponents/CurrencyDiv.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+interface Coin {
+  id: string;
+  symbol: string;
+  image: string;
+  current_price: number;
+  market_cap_change_percentage_24h: number;
+}
+
+const CurrencyDiv = ({ coin }: { coin: Coin }) => {
```
Comment (Basiol97) [inline src/app/components/homeComponents/currencySelector/currencySelectorComponents/CurrencyDiv.tsx]: No need to specify the Div in CurrencyDiv. Update the naming.

File: src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { NextPrevBtnSVG } from "../../../../../../public/currencySelectorSVGS";
+
+export const PrevArrow = ({ onClick }: { onClick: () => void }) => (
+  <button
+    className="z-20 rotate-180 absolute transform -translate-y-1/2"
+    onClick={onClick}
+    style={{ top: "45%", right: "97%" }}
```
Comment (Basiol97) [inline src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx]: why inline styling?

File: src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  >
+    <NextPrevBtnSVG />
+  </button>
+);
+
+export const NextArrow = ({ onClick }: { onClick: () => void }) => (
+  <button
+    className="z-20  absolute transform -translate-y-1/2"
+    onClick={onClick}
+    style={{ top: "55%", left: "97%" }}
```
Comment (Basiol97) [inline src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx]: again, try to keep the styling consistent

File: src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx (hunk 3 above)
Comment (TSherpa1) [reply src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx]: Trying to position them within tailwind wasn't working, noticed that it worked with inline styling

File: src/app/components/homeComponents/currencySelector/page.tsx (hunk 1 above)
Comment (TSherpa1) [reply src/app/components/homeComponents/currencySelector/page.tsx]: The dummy data you mean?

**Review by Basiol97 — APPROVED** [review]

**Pull request merged**